api/base/permissions.py

```diff
@@ -53,6 +54,12 @@ def _get_scopes(self, request, view):
             except AttributeError:
                 raise ImproperlyConfigured('TokenHasScope requires the view to define the '
                                            'required_read_scopes attribute')
+            assert is_iterable_but_not_string(view.required_read_scopes), \
+                'The required_read_scopes must be an iterable of CoreScopes'
+            if view.required_read_scopes and isinstance(view.required_read_scopes[0], tuple):
```

Comment:
The `view.required_read_scopes` will always contain `CoreScopes` from what I've seen. For example, if I set `required_read_scopes = [ComposedScopes.FOO]`, where `ComposedScopes.FOO = (CoreScopes.BAR, CoreScopes.FIZZ, )`, then `view.required_read_scopes = [(CoreScopes.BAR, CoreScopes.FIZZ)]`